src/main/java/org/isf/accounting/manager/BillBrowserManager.java

```diff
@@ -108,52 +110,56 @@ public List<BillItems> getItems(int billID) throws OHServiceException {
 
 	/**
 	 * Retrieves all the bills of a given patient between dateFrom and datTo
+	 * 
 	 * @param dateFrom
 	 * @param dateTo
 	 * @param patient
 	 * @return the bills list
-	 * @throws OHServiceException 
+	 * @throws OHServiceException
 	 */
-	public List<Bill> getBills(LocalDateTime dateFrom, LocalDateTime dateTo,Patient patient) throws OHServiceException {
+	public List<Bill> getBills(LocalDateTime dateFrom, LocalDateTime dateTo, Patient patient) throws OHServiceException {
 		return ioOperations.getBillsBetweenDatesWherePatient(dateFrom, dateTo, patient);
 	}
 
 	/**
 	 * Retrieves all the billPayments for a given patient between dateFrom and dateTo
+	 * 
 	 * @param dateFrom
 	 * @param dateTo
 	 * @param patient
 	 * @return the list of payments
-	 * @throws OHServiceException 
+	 * @throws OHServiceException
 	 */
-	public List<BillPayments> getPayments(LocalDateTime dateFrom, LocalDateTime dateTo,Patient patient) throws OHServiceException {
+	public List<BillPayments> getPayments(LocalDateTime dateFrom, LocalDateTime dateTo, Patient patient) throws OHServiceException {
 		return ioOperations.getPaymentsBetweenDatesWherePatient(dateFrom, dateTo, patient);
 	}
 
 	/**
 	 * Gets all the {@link BillPayments} for the specified {@link Bill}.
+	 * 
 	 * @param billID the bill id.
 	 * @return a list of {@link BillPayments}
-	 * @throws OHServiceException 
+	 * @throws OHServiceException
 	 */
 	public List<BillPayments> getPayments(int billID) throws OHServiceException {
 		return ioOperations.getPayments(billID);
 	}
-	
+
 	/**
 	 * Stores a new {@link Bill} along with all its {@link BillItems} and {@link BillPayments}
+	 * 
 	 * @param bill - the bill to store.
```

Comment:
Technically the `-` isn't needed as the javadoc provides it; to if it is specified here then it is doubled.
```suggestion
	 * @param bill the bill to store.
```